secure_message/application.py

```diff
@@ -83,6 +77,22 @@ def handle_exception(error):  # NOQA pylint:disable=unused-variable
     return app
 
 
+def retry_if_database_error(exception):
+    logger.error('Database error has occurred', error=exception)
+    return isinstance(exception, DatabaseError) and not isinstance(exception, ProgrammingError)
+
+
+@retry(retry_on_exception=retry_if_database_error, wait_fixed=2000, stop_max_delay=30000, wrap_exception=True)
+def create_db(app, app_config):
+    database.db.init_app(app)
+    logger.info('Starting Secure Message Service...', config=app_config)
```

Comment:
This log line can probably live outside of the create_db function.

Reply:
👍 Done